experiment/simulation/js/main.js

```diff
@@ -1 +1,243 @@
-//Your JavaScript goes in here
+'use strict';
+let TotalDrops = 0;
```

Comment:
Please follow camelCase naming convention for JS variables and functions. Names should start with a lowercase character.